apps/store/src/blocks/VideoListBlock.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from '@emotion/styled'
+import { storyblokEditable } from '@storyblok/react'
+import { mq } from 'ui'
+import { ExpectedBlockType, SbBaseBlockProps } from '@/services/storyblok/storyblok'
+import { VideoBlock, VideoBlockProps } from './VideoBlock'
+
+type VideoListBlockProps = SbBaseBlockProps<{
+  videos: ExpectedBlockType<VideoBlockProps>
+}>
+
+export const VideoListBlock = ({ blok }: VideoListBlockProps) => {
+  return (
+    <Grid numberOfCols={blok.videos.length} {...storyblokEditable(blok)}>
+      {/* This is a workaround for centralizing content in containers that may scroll. */}
+      {/* Using justify-content: center; on a scroll container will cause an issue where */}
+      {/* the first item will be cutted out. Those <span>'s will occupy flexible columns (1fr) */}
+      {/* added to the edges, which will centrilize the remaining content. */}
+      {/* There is a CSS working draft that solves this issue: justify-content: center safe; */}
+      {/* However it's not well support yet. More info --> http://bitly.ws/y7ak */}
+      <span aria-hidden="true" />
+      {blok.videos.map((nestedVideoBlock) => (
+        <ScrollableItem key={nestedVideoBlock._uid}>
+          <VideoBlock blok={nestedVideoBlock} />
+        </ScrollableItem>
+      ))}
+      <span aria-hidden="true" />
+    </Grid>
+  )
+}
+VideoListBlock.blockName = 'videoList'
+
+const Grid = styled.div<{ numberOfCols: number }>(({ theme, numberOfCols }) => ({
+  display: 'grid',
+  gridAutoFlow: 'column',
+  gridTemplateColumns: `1fr repeat(${numberOfCols}, 90vw) 1fr`,
+  gap: theme.space[4],
+  paddingInline: theme.space[6],
+  scrollSnapType: 'x mandatory',
+  overflowX: 'auto',
+  [mq.md]: {
+    gridTemplateColumns: `1fr repeat(${numberOfCols}, minmax(auto, 20.75rem)) 1fr`,
+  },
+}))
+
```

Comment:
Since the video block has padding inline when using it as a standalone block, we need to remove it when using it in a list block to make the spacing as design. :) 

(right now it has too much spacing in desktop)
<img width="1143" alt="Screenshot 2022-12-22 at 11 11 45" src="https://user-images.githubusercontent.com/6661511/209111573-6eab2bad-22d8-4fe2-a505-146608acbc3d.png">

We could export the `Wrapper` (renamed it `VideoBlockWrapper` here)  from `VideoBlock` and remove it's padding when used in a listing block.
```suggestion
  [`& ${VideoBlockWrapper}`]: {
    paddingInline: 0,
  },
```